renku_notebooks/api/classes/dataset.py

```diff
@@ -0,0 +1,181 @@
+from flask import current_app
+import boto3
+from botocore import UNSIGNED
+from botocore.client import Config
+from botocore.exceptions import EndpointConnectionError, ClientError, NoCredentialsError
+
+
+class Dataset:
```

Comment:
I understand that these are called datasets in the datashim project. However, at some point we want to distinguish them from our renku datasets. I wondering if this class could be the right point to make the "translation", by calling this class something like `S3mount` rather than dataset. wdyt?